modules/swagger-parser-v2-converter/src/main/java/io/swagger/parser/v2/SwaggerConverter.java

```diff
@@ -456,6 +456,11 @@ public Contact convert(v2.io.swagger.models.Contact v2Contact) {
         contact.setName(v2Contact.getName());
         contact.setEmail(v2Contact.getEmail());
 
+        // TODO - uncomment this code after adding extensions to v2Contact object
+        /*if (v2Contact.getVendorExtensions() != null && v2Contact.getVendorExtensions().size() > 0) {
```

Comment:
Keep the TODO but remove the commented code.

Reply:
Done!